pandas/core/series.py

```diff
@@ -4630,6 +4630,34 @@ def isin(self, values) -> "Series":
         4     True
         5    False
         Name: animal, dtype: bool
+        
+        New behaviour from pandas v1.2.0 caused some tests to fail because users where unaware of this change.
+        
+        Before v1.2.0:
+        
+        >>>import pandas as pd
+        pd.Series([0]).isin(['0']) 
+        # 0    True
+        # dtype: bool
+        pd.Series([1]).isin(['1']) 
+        # 0    True
```

Comment:
yeah just show the new example (you can add a comment that strings and integers are distinct). Keep the style in-line with the other examples.

Reply:
Ok thanks for the feedback, I will fix it.